extended/src/main/java/apoc/vectordb/VectorDbUtil.java

```diff
@@ -130,4 +134,18 @@ public static void methodAndPayloadNull(Map<String, Object> config) {
         config.put(METHOD_KEY, null);
         config.put(BODY_KEY, null);
     }
+
+    public static List addMetadataKeyToFields(Map<String, Object> config) {
+        List listFields = (List) config.getOrDefault(FIELDS_KEY, new ArrayList<>());
+
+        Map<String, Object> mapping = (Map<String, Object>) config.get(MAPPING_KEY);
+
+        String metadataKey = MapUtils.getString(mapping, METADATA_KEY);
+
+        if (StringUtils.isNotEmpty(metadataKey)) {
```

Comment:
Are you sure that the behaviour should be to always add the METADATA_KEY mapping? Or should we add them only when the FIELDS_KEY is empty?